response_operations_ui/templates/reporting-unit-survey.html

```diff
@@ -134,9 +135,6 @@ <h3 id="RESPONDENTS" class="ons-u-fs-m">Respondents</h3>
                             },
                             {
                                 "value": "Enrolment status"
-                            },
```

Comment:
If you have `messages.edit` permission, the `Create message` button still appears but there isn't a table header, and in the design it seems that this column is still present it only removes the button when the user does not have permissions

Reply:
<img width="1301" alt="Screenshot 2022-06-13 at 12 00 51" src="https://user-images.githubusercontent.com/41422777/173339954-1448be43-3345-4096-9bff-cfb9da079d2b.png">

Reply:
I'd just check with Gareth or someone to make sure how they want it